tests/conftest.py

```diff
@@ -705,7 +706,23 @@ async def enable_auth() -> bool:
 
 
 @pytest.fixture
-async def access_policy() -> str:
+def auth_method() -> AuthMethod:
```

Comment:
```suggestion
def authentication_method() -> AuthMethod:
```

to be clear which of the two auth's this is about